scripts/fetch/fetch_utils.R

```diff
@@ -0,0 +1,33 @@
+## utils for fetch functions
+
+netcdf_to_sp <- function(filepath, crs_str) {
+  
+  nc <- ncdf4::nc_open(filepath)
+  
+  cell_size <- .06 # degrees
+  
+  # generate point sp
+  x <- ncdf4::ncvar_get(nc, nc$var$lon)
+  y <- ncdf4::ncvar_get(nc, nc$var$lat)
+  coords <- data.frame(x=matrix(x, ncol = 1), y=matrix(y, ncol = 1))
+  geo_point_data <- sp::SpatialPoints(coords, proj4string = sp::CRS(crs_str))
+  geo_point_data <- sp::spTransform(geo_point_data, sp::CRS(crs_str))
```

Comment:
Note that `crs_str` would usually be different from creating the data to projecting the data. I would hard code `crs_str` on line 13 to be `epgs:4326` (wgs84 lat/lon) then use the `crs_str` input parameter as the output CRS of the raster you get back.

Reply:
as the "output CRS" you mean leave it there on line 14 so that the point data is being transformed into it?

Reply:
That's correct.